### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by genex89** [garethgeorge/backrest#280] [General]
Where can I find the documentation? All links to the wiki don't work

**Comment by garethgeorge** [comment]
Thanks for surfacing this, that's definitely unintended. Looks like some part of the build for the wiki broke. I'll look at fixing this over the weekend.

**Reply by garethgeorge** [reply]
Fixed, wiki links are now working https://garethgeorge.github.io/backrest/introduction/getting-started
